models/zeroex/base/zeroex_base_api_fills.sql

```diff
@@ -0,0 +1,264 @@
+{{  config(
+    tags=['dunesql'],
+        alias = alias('api_fills_base'),
```

Comment:
can you remove `base` in all aliases in PR?

then add `schema = 'zeroex_base'` in all PRs too?